delphi-checks/src/main/java/au/com/integradev/delphi/checks/RedundantParenthesesCheck.java

```diff
@@ -50,4 +51,15 @@ public DelphiCheckContext visit(
     }
     return super.visit(expression, context);
   }
+
+  private static boolean isRedundant(ParenthesizedExpressionNode expression) {
+    ExpressionNode parenthesized = expression.getExpression();
+
+    if (parenthesized instanceof PrimaryExpressionNode) {
+      return !(ExpressionNodeUtils.isInherited(parenthesized)
+          && !ExpressionNodeUtils.isBareInherited(parenthesized));
```

Comment:
Do you think this might be more clear de Morgan'd?

Reply:
Pulled it out into a local static method, what do you think?

Reply:
I think that makes sense.